locust/user/task.py

```diff
@@ -49,6 +49,54 @@ def my_task()
     else:
         return decorator_func
 
+def tag(tag_name=None):
+    """
+    Decorator for tagging tasks and TaskSets with the given tag name. You can then limit
+    the test to only execute tasks that are tagged with any of the tags provided by the
+    --tags command-line argument. Example::
+
+        class ForumPage(TaskSet):
+            @tag('thread')
+            @task(100)
+            def read_thread(self):
+                pass
+
+            @tag('thread')
+            @tag('post')
+            @task(7)
+            def create_thread(self):
+                pass
+
+            @tag('post')
+            @task(11)
+            def comment(self):
+                pass
+    """
+
+    def decorator_func(decorated):
+        if issubclass(type(decorated), TaskSetMeta):
```

Comment:
Is there any reason you check for TaskSetMeta instead of TaskSet?

Reply:
Well, the type of a subclass of TaskSet is TaskSetMeta, which isn't a subclass of TaskSet. I would like to get rid of the `type()` though if I could. Doing something like `issubclass(decorated, TaskSet)` would work if it didn't throw an error when `decorated` is a function. 

I think I'll change this (and the similar check in filter_tasks_by_tags) to `hasattr(decorated, 'tasks')`

Reply:
> I think I'll change this (and the similar check in filter_tasks_by_tags) to hasattr(decorated, 'tasks')

Sounds good!